overrides/netlify.toml

```diff
@@ -143,5 +146,6 @@
         https://staging.ask.gov.sg
         wss://*.zendesk.com
         wss://*.zopim.com
+        https://*.dcube.cloud
```

Comment:
should we add `font-src` above too?

Reply:
I think there's no need for `font-src` - we already have a universal whitelist for that

Reply:
okay got it